From 5e5f77541882849e9db3dd3517e2fc0100cc7706 Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Fri, 14 Aug 2009 16:12:32 +0200
Subject: [PATCH 07/33] fixed missusage of contains_outof

Fixed missusage of contains_outof, use it only to match
out of multiple cases.
---
 fdi/information/10freedesktop/10-modem.fdi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/fdi/information/10freedesktop/10-modem.fdi b/fdi/information/10freedesktop/10-modem.fdi
index c32e360..1f3ac50 100644
--- a/fdi/information/10freedesktop/10-modem.fdi
+++ b/fdi/information/10freedesktop/10-modem.fdi
@@ -531,7 +531,7 @@
             <remove key="info.capabilities" type="strlist">net.80203</remove>
       </match>
     </match>
-    <match key="info.parent" contains_outof="bdb_1904">
+    <match key="info.parent" contains="bdb_1904">
       <match key="info.parent" suffix="_if6">
             <remove key="info.capabilities" type="strlist">net.80203</remove>
       </match>
-- 
1.7.1

